blocks/eda-frontend/src/components/SchematicEditor/SimulationScreen.js

```diff
@@ -743,9 +749,32 @@ export default function SimulationScreen ({ open, close }) {
                           : <div />
                       }
                       </>
-                    : (result.isGraph === 'true') ? <span>{typography1}</span> : <span />
+                    : (result.isGraph === 'false') ? <span>{typography1}</span> : <span />
+                }
+                {/* Diplay of Simulation parameter Not present */}
+                {
+                  (error === 1)
+                    ? <Grid item xs={12} sm={12}>
+                    <Paper className={classes.paper}>
+                      <Typography variant='h4' align='center' gutterBottom>
+                        {typography1}
```

Comment:
```suggestion
                        {error}
```
Change to the actual error message